browserstack.yml

```diff
@@ -1,12 +1,12 @@
 userName: BROWSERSTACK_USERNAME
-accessKey: BROWSERSTACK_ACCESS_KEY
+accessKey: BROSWSERSTACK_ACCESS_KEY
```

Comment:
Error is variable spelling, should be BROWSERSTACK_ACCESS_KEY

Reply:
Have updated the variable spelling